backend/app/views.py

```
@@ -596,14 +643,25 @@ def index_view(request, instance=None):
         else:
             logger.debug("Retrieved aggregated data across all portal instances")
 
+        table_columns = {}
+        for name, cfg in TABLE_VIEW_MODEL_CONFIG.items():
+            tc = cfg["table_class"]
+            defaults = list(tc.default_visible_columns())
+            table_columns[name] = {
+                "columns": [{"name": n, "label": l, "selected": n in defaults}
+                            for n, l in tc.get_column_labels()],
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)